packages/elements/src/slider/index.ts

```diff
@@ -1197,17 +1270,17 @@ export class Slider extends ControlElement {
   private renderNumberField (value: string, name: string): TemplateResult {
     return html`
       <ef-number-field
-        @blur=${this.onBlur}
-        @keydown=${this.onKeydown}
+        @blur=${this.onNumberFieldBlur}
+        @keydown=${this.onNumberFieldKeyDown}
         part="input"
         name="${name}"
         no-spinner
+        aria-label="${name} input"
```

Comment:
`input`? Does it need to be translated?

Reply:
Yes maybe both `{from, to, value} input`, but as a temporal solution to labelling inputs and thumbs I'm not sure if it is worth. Or otherwise we shouldn't label them at all. What do you think ?

Reply:
Yeah I agree, it is a strange design choice. I'm going to remove label for now.